tools/cli/NearObjectCli.cxx

```diff
@@ -105,27 +105,24 @@ NearObjectCli::CreateParser() noexcept
 namespace detail
 {
 /**
- * @brief Helper function to produce a mapping of enumeration strings to their
- * corresponding field names.
- *
- * This is used for CLI11 CheckedTransformer which validates string inputs and
- * maps them to their corresponding enumeration value.
+ * @brief Helper function to produce a mapping of enumeration field names to their
+ * corresponding values.
  *
  * @tparam EnumType The type of the enumeration.
- * @return std::map<std::string, EnumType>
+ * @return std::map<std::string, int>
```

Comment:
Looks like this comment needs updating (should say `unordered_map` here instead of `map`).